.changeset/stupid-buckets-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kadena/react-ui': patch
+---
+
+small fixes for the accordion component
```

Comment:
Changsets will end up making up the release notes so I recommend writing them with more detail/presentation for consumers in mind. For example: 
`Added expand animation to the Accordion component`
...whatever else you think consumers would be important to communicate